src/libraries/System.Reflection.Emit.ILGeneration/tests/ILGenerator/EmitMethodInfo.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Reflection.Emit.Tests
+{
+    public interface IWithIn<T>
+    {
+        void Method(in int arg);
+    }
+    public class ILGeneratorEmitMethodInfo
+    {
+        [Fact]
+        public void EmitMethodInfo()
+        {
+            var methodType = typeof(IWithIn<int>);
+            var method = methodType.GetMethod("Method");
```

Comment:
Nit: We use `var` only when the type is obvious from the right side.

https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md